### PR TITLE
Add sync webhooks for tax calculations

### PR DESCRIPTION
**Pull request by Droniu**
I want to merge this change because it
- adds sync webhooks for tax calculations
- adds badges to multi select fields
- reduces z-index on multi select fields

**PR intended to be tested with API branch:**  main

### Screenshots

image

### Pull Request Checklist

1. [x] This code contains UI changes
2. [ ] All visible strings are translated with proper context including data-formatting
3. [ ] Attributes `[data-test-id]` are added for new elements
4. [ ] Changes are mentioned in the changelog
5. [ ] The changes are tested in different browsers and in light/dark mode

### Test environment config

API_URI=https://automation-dashboard.staging.saleor.cloud/graphql/
MARKETPLACE_URL=https://marketplace-gray.vercel.app/

### CODE REVIEW

**Comment by patrys** [comment]
Task linked: [SALEOR-7639 Add flags for tax calculation in sync webhooks](https://app.clickup.com/t/2549495/SALEOR-7639)

File: src/webhooks/utils.tsx [hunk 1]
```diff
@@ -1,20 +1,29 @@
...
 
 export function isUnnamed(webhook: WebhookFragment): boolean {
   return ["", null].includes(webhook?.name);
 }
 
+const isWebhookInPreview = (
+  webhook: WebhookEventTypeSyncEnum | WebhookEventTypeAsyncEnum,
+) =>
+  webhook === WebhookEventTypeSyncEnum.CHECKOUT_CALCULATE_TAXES ||
+  webhook === WebhookEventTypeSyncEnum.ORDER_CALCULATE_TAXES;
```
Comment (dominik-zeglen) [inline src/webhooks/utils.tsx:18]: A bit more scalable  ;)

```suggestion
[WebhookEventTypeSyncEnum.CHECKOUT_CALCULATE_TAXES, WebhookEventTypeSyncEnum.ORDER_CALCULATE_TAXES].includes(webhook);
```

**Review by dominik-zeglen — APPROVED** [review]

**Comment by patrys** [comment]
Is there a way to make the entire event name readable? With the preview label, it's even harder to use.

**Comment by timuric** [comment]
> Is there a way to make the entire event name readable? With the preview label, it's even harder to use.

yes, we are working on it

**Comment by Droniu** [comment]
@patrys  It was fixed already, I didn't replace the screenshot. The view is in uniform grid now, so more space for webhook names: 
image

**Review by orzechdev — APPROVED** [review]

**Pull request merged**